latex/chain.tex

```diff
@@ -694,10 +931,13 @@ \subsection{Block Body Transition}
       \var{txs} \leteq \bbody{block}
       &
       \var{vk} \leteq \bissuer{bhb}
+      &
+      \var{hk} \leteq \hashKey{vk}
       \\
-      \bSize{bh} < \fun{maxBBSize}~\var{pp}
+      \bBodySize{txs} < \fun{maxBBSize}~\var{pp} % TODO is bBodySize the right
+                                % thing here?
       &
-      \var{vk}\mapsto n\in (b \unionoverrideLeft \{\var{vk}\mapsto 0\})
+      \var{hk}\mapsto n\in (b \unionoverrideLeft \{\var{vk}\mapsto 0\})
       \\~\\
```

Comment:
It looks like we are still missing a signature check here. we can get `\sigma := bsig bhb`, and then verify the `\sigma` with `vk` against `txs`.  we can do this now if it's convenient, or I'll do it next.

Reply:
added the signature check for `txs`